cw/messaging.py

```diff
@@ -1,19 +1,26 @@
 import pika
 
 
-def open_connection (cfg):
-    return pika.BlockingConnection(
-        pika.ConnectionParameters(host=cfg['rabbit-host'],
-                                  port=cfg['rabbit-port'],
-                                  ssl=cfg['rabbit-ssl']))
+class MessagingException(Exception):
```

Comment:
messagingexception should be further specialized to describe the real situation in order to add more value.

OpeningConnectionException... etc.  That way you can ```except``` for only that and do something about it further up.  no reason it can't dereive from MessagingException though.